lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_EnsembleCopulaCouplingUtilities.py

```diff
@@ -534,16 +496,18 @@ def test_percentile_is_dimension_coordinate_multiple_timesteps(self):
         cube = set_up_cube(data, "air_temperature", "degreesC",
                            timesteps=2, x_dimension_length=2,
                            y_dimension_length=2)
-        cube.coord("realization").rename("percentile")
-        cube.coord("percentile").points = np.array([0.1, 0.5, 0.9])
+        cube.coord("realization").rename("percentile_over_realization")
+        cube.coord("percentile_over_realization").points = (
+            np.array([0.1, 0.5, 0.9]))
```

Comment:
This runs either way, but should these percentiles be *100 as in other routines?